### PR TITLE
ED-3610 jira to geckboard script

### PR DESCRIPTION
**Pull request by kowalcj0**
This [ticket](https://uktrade.atlassian.net/browse/ED-3610)

### CODE REVIEW

File: scripts/jira_to_geckoboard.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    for issue in issues:
+        labels = [label for label in issue['fields']['labels']
+                  if label.startswith(label_prefix)]
+        for label in labels:
+            if remove_label_prefix:
+                label = label.replace(label_prefix, '')
+            if ignored_labels:
+                if label in ignored_labels:
+                    continue
+            if look_for:
```
Comment (sdonk) [inline scripts/jira_to_geckoboard.py]: You don't need a double if here

Comment (kowalcj0) [reply scripts/jira_to_geckoboard.py]: I do need it, cos `look_for` is an optional argument. To get rid of it I'd have to set its default value to `[]`

File: scripts/jira_to_geckoboard.py [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+JIRA_HOST = os.environ['JIRA_HOST']
+JIRA_USERNAME = os.environ['JIRA_USERNAME']
+JIRA_PASSWORD = os.environ['JIRA_PASSWORD']
+
+# Instantiate clients
+JIRA_CLIENT = JiraClient(JIRA_HOST, basic_auth=(JIRA_USERNAME, JIRA_PASSWORD))
+GECKO_CLIENT = GeckoClient(GECKOBOARD_API_KEY)
+
+# other variables
+TODAY = str(date.today())
```
[the remark this answers is not in the document]
Comment (kowalcj0) [reply scripts/jira_to_geckoboard.py]: it's so ugly @sdonk `date.isoformat(datetime.date.today())`

[the remark this answers is not in the document]
Comment (sdonk) [reply scripts/jira_to_geckoboard.py]: @kowalcj0 you can use `date.today().isoformat()` :D

File: scripts/jira_to_geckoboard.py (hunk 1 above)
Comment (sdonk) [reply scripts/jira_to_geckoboard.py]: defaulting to a not mutable object will cause a lot of fun, leave the two ifs

**Pull request merged**